src/order/category/Frame.lean

```diff
@@ -56,3 +58,9 @@ instance has_forget_to_Lattice : has_forget₂ Frame Lattice :=
   inv_hom_id' := by { ext, exact e.apply_symm_apply _ } }
 
 end Frame
+
+/-- The forgetful functor from `Topᵒᵖ` to `Frame`. -/
```

Comment:
```suggestion
/-- The functor from `Topᵒᵖ` to `Frame`, sending a topological space to its frame of open subsets. -/
```
I wouldn't call this a forgetful functor, even if I understand why you did. It doesn't fit in the framework of concrete categories and forgetful functors like `forget₂`.

Reply:
Neither does `Top_to_Locale`, which is why it's not a `has_forget₂` instance, but it's definitely called a forgetful functor nevertheless.

Reply:
Hmm, I find it funny to call a contravariant functor "forgetful". Clearly it's not just forgetting something. Anyway, I don't care too much about this issue.

Reply:
Oh sorry my comment was mostly about `Top_to_Locale` which is a forgetful functor but not in the `forget₂` sense.